messages/bg.json

```
@@ -0,0 +1,131 @@
+  "omnishipping.pickupOption": "Опция за получаване",
+  "omnishipping.findPickup": "Открийте място за получаване в магазин близо до вас.",
+  "omnishipping.noPickupSelected": "Все още не сте избрали магазин",
+  "omnishipping.pickupReceiver": "Получател на доставката",
+  "omnishipping.slaOption": "{name} - {price} - {shippingEstimate}",
+  "omnishipping.slaOptionSimple": "{shippingEstimate}{scheduled}",
+  "omnishipping.shippingEstimateScheduled": "Планирана дата",
+  "omnishipping.shippingEstimate-bd": "{timeAmount, plural, =0 {днес} one {след # работен ден} other {след максимум # работни дни}}",
+  "omnishipping.shippingEstimate-d": "{timeAmount, plural, =0 {днес} one {след # ден} other {след максимум # дни}}",
+  "omnishipping.shippingEstimate-h": "{timeAmount, plural, =0 {Точно сега} one {след # час} other {до # часа}}",
+  "omnishipping.shippingEstimate-m": "{timeAmount, plural, =0 {Точно сега} one {след # минута} other {до # минути}}",
+  "omnishipping.shippingEstimate-db": "Работни дни",
+  "omnishipping.free": "Безплатно",
+  "omnishipping.shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Готово днес} one {Готово след # работен ден} other {Готово след # работни дни}}",
+  "omnishipping.shippingEstimatePickup-d": "{timeAmount, plural, =0 {Готово днес} one {Готово след # ден} other {Готово след # дни}}",
+  "omnishipping.shippingEstimatePickup-h": "{timeAmount, plural, =0 {Готово за получаване} one {Готово след # час} other {Готово след # часа}}",
+  "omnishipping.shippingEstimatePickup-m": "{timeAmount, plural, =0 {Готово за получаване} one {Готово след # минута} other {Готово след # минути}}",
+  "omnishipping.scheduled": "Планирано",
+  "omnishipping.scheduledDeliveryChoice": "Изберете дата за доставка",
+  "omnishipping.scheduledDeliverySplitDisclaimer": "Доставката на някои продукти не може да се планира. Изберете метод на доставка за тяхното получаване.",
+  "omnishipping.scheduledDeliveryPartial": "{amount, plural, =0 { } one {Планиране на # продукт} other {Планиране на # продукти}}",
+  "omnishipping.scheduledDeliveryPrice": "Цени от {price}",
+  "omnishipping.scheduledDeliveryAll": "Планиране на всички",
+  "omnishipping.deliveryOptionsLeft": "Начин на доставка за останалите продукти",
+  "omnishipping.selectPickupPoint": "Изберете място за получаване в магазин ",
+  "omnishipping.selectDeliveryOptions": "Изберете възможности за доставка",
+  "omnishipping.pointDetails": "Подробности за магазина за получаване",
+  "omnishipping.closeButton": "затваряне",
+  "omnishipping.deliveryDate": "Планиране",
+  "omnishipping.deliveryTime": "Дата за доставка",
+  "omnishipping.datepickerDateFormat": "dddd, LL",
+  "omnishipping.deliveryWindows": "От {startDate} до {endDate}",
+  "omnishipping.deliveryWindowsSummary": "{date}, между {startDate} и {endDate}",
+  "omnishipping.deliveryWindowsSummaryDate": "{date}, между {startDate} и {endDate}",
+  "omnishipping.deliveryWindowsSummarytime": "между {startDate} и {endDate}",
```

This is a good key to refactor in the entire project if you want.
```suggestion
  "omnishipping.deliveryWindowsSummaryTime": "между {startDate} и {endDate}",
```